src/frontend/model/throw_error.rs

```diff
@@ -0,0 +1,7 @@
+//replace this with tracing levels as soon as arbitrary const types get stable
+
+pub const THROW_ERROR_SOFT: u8 = 0;
+pub const THROW_ERROR_HARD: u8 = 1;
+
+#[derive(Default, Clone)]
+pub struct ThrowError<const L: u8>(pub Vec<String>);
```

Comment:
I guess this works, but wouldn't an enum be better?

Reply:
And if we just don't use a const parameter?

Reply:
then we cant distinguish them :/ / see below